packages/block-editor/src/components/inspector-controls-tabs/use-inspector-controls-tabs.js

```diff
@@ -46,12 +47,23 @@ export default function useInspectorControlsTabs( blockName ) {
 		typography: typographyGroup,
 	} = InspectorControlsGroups;
 
+	const showIconLabels = useSelect( ( select ) => {
+		return select( preferencesStore ).get(
+			'core/edit-post',
```

Comment:
I don't believe there should be such a dependency in `block editor` package. In other cases the labels are added through css([example](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/rich-text/style.scss#L33)).

Reply:
Thanks @ntsekouras 👍 

I agree regarding not introducing the dependency but put this draft up to use in a conversation looking for alternate solutions. The CSS example is just what I was looking for. Cheers 🍻

Reply:
✅ Done in https://github.com/WordPress/gutenberg/pull/46968/commits/02bde2528cb549f9e76c8e4925f449dd74966646